geowebcache/web/src/main/webapp/WEB-INF/geowebcache-rest-context.xml

```diff
@@ -7,7 +7,6 @@
     http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd
     http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc-4.0.xsd">
 
-  <mvc:annotation-driven />
```

Comment:
How is anything working without this ?!

Reply:
GeoServer is in the same situation, the rest module doesn't have this tag in it's context.xml.